slf4j/src/test/scala/zio/logging/slf4j/Slf4jLoggerTest.scala

```diff
@@ -12,25 +12,24 @@ import scala.jdk.CollectionConverters._
 
 object Slf4jLoggerTest extends DefaultRunnableSpec {
 
+  val uuid1                     = UUID.randomUUID()
   val logLayer: ULayer[Logging] =
     Slf4jLogger.makeWithAnnotationsAsMdc(
-      List(LogAnnotation.CorrelationId, LogAnnotation.Level)
+      mdcAnnotations = List(LogAnnotation.CorrelationId, LogAnnotation.Level),
+      initialContext = LogContext.empty.annotate(LogAnnotation.CorrelationId, Some(uuid1))
     )
 
   def spec =
     suite("slf4j logger")(
       testM("test1") {
         for {
-          uuid1 <- UIO(UUID.randomUUID())
           uuid2 <- UIO(UUID.randomUUID())
-          _     <- log.locally(_.annotate(LogAnnotation.CorrelationId, Some(uuid1))) {
```

Comment:
why this has changed?

Reply:
Because this annotation has been moved to the initial context to show that `uuid1` is still applied.

Reply:
should we add another test ?